vars/common.groovy

```diff
@@ -97,3 +99,17 @@ void throwErrorIfStringIsEmpty(def variable, String error_message = "Variable is
     error(error_message)
   }
 }
+
+static void configureEurekaTokenLifeSpan(RancherNamespace ns, String lifespan = "3600") {
+  def keycloakUrl = "https://${ns.generateDomain('keycloak')}"
+  RestClient client = new RestClient(this)
+  Map headers = ['Content-Type': 'application/x-www-form-urlencoded']
+  def body = "grant_type=password&username=admin&password=SecretPassword&client_id=admin-cli"
```

Comment:
Here I can see some hardcoded values.
Can we move them to https://github.com/folio-org/pipelines-shared-library/blob/master/src/org/folio/Constants.groovy?